apidump/apidump.go

```diff
@@ -100,6 +100,9 @@ type Args struct {
 
 	// Print packet capture statistics after N seconds.
 	StatsLogDelay int
+
+	// Periodically report telemetry every N seconds thereafter
+	TelemetryInterval int
```

Comment:
Why not use a `time.Duration`? (Seems like `StatsLogDelay` should also have been a `time.Duration`.)